tests/utils/test_request_helper.py

```diff
@@ -0,0 +1,9 @@
+from workos.utils.request_helper import RequestHelper
+
+
+class TestRequestHelper:
+
+    def test_build_parameterized_url(self):
+        assert RequestHelper.build_parameterized_url("a/b/c") == "a/b/c"
+        assert RequestHelper.build_parameterized_url("a/{b}/c", b="b") == "a/b/c"
+        assert RequestHelper.build_parameterized_url("a/{b}/c", b="test") == "a/test/c"
```

Comment:
... what happens if `RequestHelper.build_parameterized_url("a/{b}/c", b="i/am/being/sneaky")`?

Reply:
That's allowed, currently and would output `a/i/am/being/sneaky/c`.